transit_odp/dqs/views/suppress_observation.py

```diff
@@ -16,10 +17,21 @@ def suppress_observation(self, request, **kwargs):
         """Action to suppress the observation result"""
 
         request_data = request.data
+        session_data = request.session
+        org_id = request_data.get("organisation_id", None)
+
+        if session_data and session_data.get("_auth_user_id", None):
```

Comment:
None shouldn't be defined as default for this value in this case

Reply:
done